dev/com.ibm.ws.jsfContainer_fat_2.3/bnd.bnd

```diff
@@ -54,4 +54,6 @@ fat.project: true
 	io.openliberty.org.apache.commons.logging;version=latest,\
 	io.openliberty.org.apache.commons.codec;version=latest,\
 	net.sourceforge.htmlunit:htmlunit;version=2.20,\
+	io.openliberty.org.apache.commons.logging;version=latest,\
+	io.openliberty.org.apache.commons.codec;version=latest,\
```

Comment:
This part is already in integration.  Added locally to get the tests to pass. Not sure why Git still shows this.

Reply:
https://github.com/OpenLiberty/open-liberty/pull/17415

Reply:
This is odd can you try and fix this? It basically shows a duplicate now.